lms/.gitignore

```diff
@@ -0,0 +1,2 @@
+Cargo.lock
+target/
```

Comment:
This whole file can be removed as the toplevel `.gitignore` should address this (and also we check in `Cargo.lock` to have a deterministic build, which is the new recommendation from the Cargo team)